.github/workflows/maintenance.yml

```
@@ -0,0 +1,229 @@
+
+      - name: Update documentation dependencies
+        run: |
+          echo "Checking documentation tools..."
+
+          # Check for markdownlint updates
+          echo "Current markdownlint config:"
+          cat .markdownlint.json
+
+      - name: Create update PR
+        if: steps.actions_updates.outputs.updates_available == 'true' || steps.system_deps.outputs.updates_available == 'true'
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.10)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)